test/webserv/integration/test_connection.py

```
@@ -0,0 +1,151 @@
+        con.connect()
+
+        # request送信
+        con.sock.send(
+            b"GET / HTTP/1.1\r\nHost: localhost\r\nConnection: keep-alive\r\n\r\n"
+        )
+
+        # serverからのresponseを読み込む
+        response = receive_with_timeout(con.sock)
+        if response is None:
+            # timeout秒経過していたらNG
+            assert False
```

Ruff

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)